omni-relayer/src/startup/near.rs

```diff
@@ -0,0 +1,90 @@
+use anyhow::{Context, Result};
+use futures::StreamExt;
+use log::info;
+
+use near_crypto::InMemorySigner;
+use near_jsonrpc_client::JsonRpcClient;
+use near_lake_framework::{LakeConfig, LakeConfigBuilder};
+
+use crate::{config, utils};
+
+pub fn create_signer() -> Result<InMemorySigner> {
+    info!("Creating NEAR signer");
+
+    let account_id = std::env::var("NEAR_ACCOUNT_ID")
+        .context("Failed to get `NEAR_ACCOUNT_ID` env variable")?
+        .parse()?;
+
+    let private_key = std::env::var("NEAR_PRIVATE_KEY")
+        .context("Failed to get `NEAR_PRIVATE_KEY` env variable")?
+        .parse()?;
```

Comment:
Add the ability to pass the path to near_credential and extract the key from it.